controllers/result.go

```diff
@@ -0,0 +1,44 @@
+package controllers
+
+import (
+	"net/http"
+
+	"go-google-scraper-challenge/services/scraper"
+
+	"github.com/beego/beego/v2/server/web"
+)
+
+// ResultController operations for User
+type ResultController struct {
+	BaseController
+}
+
+// URLMapping map user controller actions to functions
+func (c *ResultController) URLMapping() {
+	c.Mapping("List", c.List)
+	c.Mapping("Create", c.Create)
+}
+
+func (c *ResultController) List() {
+	c.EnsureAuthenticatedUser(true)
+
+	c.Layout = "layouts/default.html"
```

Comment:
I think we can move it to the base controller so that we don't need to set it every child controller